docs/application-developers-reference-guide.md

```diff
@@ -0,0 +1,83 @@
+# Application Developer's Reference Guide
+
+This document is intended for cloud-native application developers targeting application deployments on Bitnami Kubernetes Production Runtime (BKPR). It describes the configuration you need to perform for automating the ingress, TLS, logging and monitoring support for their application.
```

Comment:
they need to perform

"ingress" -> should be Ingress? If yes, please change throughout.